pkg/ingestor/ingestcli/interface.go

```diff
@@ -42,14 +41,8 @@ type IngestRequest struct {
 // you can call Write multiple times to write data in a stream way before close.
 // when close, the server will return the info of SSTs generated by the server.
 type WriteClient interface {
-	// Write KV to storage layer.
-	Write(ctx context.Context, in *WriteRequest) error
-	// CloseAndRecv the client, get the response and close the client.
-	// the response should contain the SST info.
-	// we name it this way to mimic GRPC stream, and to avoid confusion with normal
-	// Close method which should always be called after creation, but in here,
-	// it's not a must.
-	CloseAndRecv(ctx context.Context) (*WriteResponse, error)
+	WriteChunk(*WriteRequest) error
```

Comment:
there is plan to unify first-gen tidb api into the same interface, in that case, there's no concept of `chunk`